app/models/cloud_volume.rb

```diff
@@ -23,6 +23,9 @@ class CloudVolume < ApplicationRecord
   has_many   :vms, :through => :hardwares, :foreign_key => :vm_or_template_id
   has_many   :volume_mappings, :dependent => :destroy
 
+  supports_not :backup_create, :description  => 'CloudVolume backup creation'
+  supports_not :backup_restore, :description => 'CloudVolume backup restore'
```

Comment:
When a subclass overrides this to say that it _does_ support it, how does that work...does the caller have to specify the description again?

Reply:
I might be missing it....but where is the description used?

It is set to `queryable_features[feature] = description` but that's private class method and I only see callers checking for the key like `queryable_features.key?(feature.to_sym)`

Reply:
I thought it was used in error messages, but maybe not?

Reply:
Correct - if we choose to keep these descriptions, most of which seem useless to me, the intention would be to override the description only if the stored value is nil.

Reply:
> I thought it was used in error messages, but maybe not?

I don't think so...I don't see `QUERYABLE_FEATURES` referenced anywhere except by `.key?(feature)`

Reply:
> I might be missing it....but where is the description used?

I do not think it is used.  See [explanation](https://github.com/ManageIQ/manageiq/pull/21046#pullrequestreview-589454705)

Reply:
:scissors:

Reply:
Removed!